swiftsimio/reader.py

```diff
@@ -345,6 +345,7 @@ def load_particle_types(self):
         for particle_type, particle_name in zip(
             self.present_particle_types, self.present_particle_names
         ):
+            print(particle_type, particle_name)
```

Comment:
Please remove print statement

Reply:
ugh, don't know why I missed that. Done now